jobs/integr8ly/ocp3/nightly/osd-testing-nightly-trigger.yaml

```diff
@@ -0,0 +1,147 @@
+                                [$class: 'StringBinding', credentialsId: "rhmi-qe1-sso-client-secret", variable: 'SSO_SECRET']
+                            ]) {
+                            build(job: 'osd-cluster-integreatly-install', propagate: false, parameters: [
+                                string(name: 'clusterName', value: "${CLUSTER_NAME}"),
+                                string(name: 'installationGitBranch', value: "${INSTALLATION_BRANCH}"),
+                                string(name: 'routerShard', value: "${ROUTER_SHARD}"),
+                                string(name: 'openShiftToken', value: "${OPENSHIFT_TOKEN}"),
+                                string(name: 'githubClientId', value: "${GH_CLIENT_ID}"),
+                                string(name: 'githubClientSecret', value: "${GH_CLIENT_SECRET}"),
+                                string(name: 'customerAdminPassword', value: "${CUSTOMER_ADMIN_PASSWORD}"),
+                                string(name: 'threeScaleAwsAccessKey', value: "${AWS_ACCESS_KEY_ID}"),
+                                string(name: 'threeScaleAwsSecretKey', value: "${AWS_SECRET_ACCESS_KEY}"),
+                                string(name: 'threeScaleBucketName', value: "${THREESCALE_BUCKET_NAME}"),
+                                string(name: 'threeScaleBucketRegion', value: "${THREESCALE_BUCKET_REGION}"),
+                                string(name: 'backupAwsKey', value: "${AWS_ACCESS_KEY_ID}"),
+                                string(name: 'backupAwsSecret', value: "${AWS_SECRET_ACCESS_KEY}"),
+                                string(name: 'backupsBucketName', value: "${BACKUPS_BUCKET_NAME}"),
+                                string(name: 'clientSecret', value: "${SSO_SECRET}")
+                            ])
+                        }
+                    } // stage
+
+                    stage ('Create test user') {
+                        build(job: 'sso-user-create-tests', propagate: false, parameters: [
+                            string(name: 'REPOSITORY', value: "${TEST_SUITES_REPOSITORY}"),
+                            string(name: 'BRANCH', value: "${TEST_SUITES_BRANCH}"),
+                            string(name: 'CUSTOMER_ADMIN_USERNAME', value: "customer-admin"),
+                            string(name: 'CUSTOMER_ADMIN_PASSWORD', value: "${CUSTOMER_ADMIN_PASSWORD}"),
+                            string(name: 'SSO_URL', value: "${SSO_URL}"),
+                            string(name: 'USERNAME', value: "${USERNAME}"),
+                            string(name: 'NUMBER_OF_USERS', value: "1"),
+                            booleanParam(name: 'CLEAN_RESOURCES', value:  Boolean.valueOf("false")),
+                        ])
+                    } // stage
+
+                /**    stage ('Run tests') {
```

Comment:
Test stage is temporarily excluded, because we don't have a working admin account in `oc`. We need either a permanent admin account in `oc` (there's a ticket for that and this is probably the easier solution) or to re-create an admin in SSO and delete it in `oc` and login via SSO to cluster to recreate the user in `oc`.

Reply:
why `/**` instead of just `/*` ? Probably no harm though.

Reply:
Old Java habit I guess.... :)